drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -28,6 +28,7 @@ testTools=Test Tools
 testEditor=Test Editor
 scenarioCheatSheet=Scenario Cheatsheet
 runScenarioSimulation=Run Test
+insertValue=(Insert value)
```

Comment:
Are we sure we want to use the brackets and second question is, are we sure we want to add them as part of the String here?
Just checking the brackets are actually needed.